common/python-copr-common.spec

```diff
@@ -16,7 +16,7 @@
 %endif
 
 Name:       python-copr-common
-Version:    0.20.1.dev1
+Version:    1.130
```

Comment:
Weird ... :-) this seems like cross-package version leak

Reply:
Ups, I didn't want to commit this...;)

Reply:
Ok, that was tito who committed that... I should be more careful about playing with that..;)